src/view_dna_n_to_random.hpp

```diff
@@ -68,6 +70,7 @@ struct dna_n_to_random_fn
 namespace seqan3::views
 {
 
-inline constexpr auto dna_n_to_random = seqan3::views::deep{dna_n_to_random_fn{}};
+template<typename TAlph = seqan3::dna4>
```

Comment:
```suggestion
template <typename TAlph = seqan3::dna4>
```

Just this minor remark, feel free to merge after fixing it!